ui/src/containers/Layout.js

```diff
@@ -143,14 +127,46 @@ const Layout = props => {
     },
   ];
 
+  const filterLanguage = languages.filter(lang => lang.name !== language);
+
+  const rightActions = [
+    {
+      type: 'dropdown',
+      text: language,
+      icon: <i className="fas fa-globe" />,
+      items: filterLanguage,
+    },
+    applications == null
+      ? {}
+      : {
+          type: 'dropdown',
+          icon: <i className="fas fa-th" />,
+          items: applications,
+        },
```

Comment:
please move this check outside this array and insert 
```
{
  type: 'dropdown',
  icon: <i className="fas fa-th" />,
  items: applications,
 }
```
only if `applications && applications.length`